gitlab/assets/dashboards/overview.json

```diff
@@ -1,1611 +1,1611 @@
-                        },
-                        "layout": {
-                            "x": 0,
-                            "y": 0,
-                            "width": 5,
-                            "height": 3
-                        }
-                    },
+                      "formula": "query1"
+                    }
+                  ],
+                  "queries": [
                     {
-                        "id": 607314244154516,
-                        "definition": {
-                            "type": "note",
-                            "content": "**Useful Links**\n\n* [Integration Docs](https://docs.datadoghq.com/integrations/gitlab/?tab=host)\n* [CI Pipeline Visibility](https://app.datadoghq.com/ci/getting-started)",
-                            "background_color": "white",
-                            "font_size": "16",
-                            "text_align": "left",
-                            "vertical_align": "top",
-                            "show_tick": false,
-                            "tick_pos": "50%",
-                            "tick_edge": "left",
-                            "has_padding": true
-                        },
-                        "layout": {
-                            "x": 0,
-                            "y": 3,
-                            "width": 5,
-                            "height": 2
-                        }
+                      "name": "query1",
+                      "data_source": "metrics",
+                      "query": "sum:gitlab.user_session_logins.count{*}.as_count()"
                     }
-                ]
+                  ],
+                  "response_format": "timeseries",
+                  "style": {
+                    "palette": "dog_classic",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "line"
+                }
+              ]
+            },
+            "layout": {
+              "x": 3,
+              "y": 4,
+              "width": 4,
+              "height": 3
+            }
+          }
+        ]
+      },
+      "layout": {
+        "x": 5,
+        "y": 0,
+        "width": 7,
+        "height": 8
+      }
+    },
+    {
+      "id": 8025458456001434,
+      "definition": {
+        "title": "Load and Project Information",
+        "background_color": "vivid_orange",
+        "show_title": true,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 1042082565617324,
+            "definition": {
+              "type": "note",
+              "content": "Load metrics inform on the volume, type, and rate of incoming HTTP requests and their routing to the appropriate controllers/actions within GitLab. Project metrics report on repository creation and pushed branches, informing on the specifics of your project progress rather than the system health itself. \n",
```

Comment:
```suggestion
              "content": "Load metrics show the volume, type, and rate of incoming HTTP requests and their routing to the appropriate controllers or actions within GitLab. Project metrics show repository creation and pushed branches, indicating the specifics of your project progress rather than the system health itself. \n",
```